dev/archery/archery/benchmark/core.py

```diff
@@ -15,21 +15,18 @@
 # specific language governing permissions and limitations
 # under the License.
 
-import pandas as pa
-
 
 class Benchmark:
     def __init__(self, name, unit, less_is_better, values, stats=None):
         self.name = name
         self.unit = unit
         self.less_is_better = less_is_better
-        self.values = pa.Series(values)
-        self.statistics = self.values.describe()
+        self.values = sorted(values)
+        self.median = self.values[len(values)/2]
```

Comment:
Need to use integer division (the `//` operator).

Reply:
Also, this isn't a very good formula for the median... You could fix it yourself (distinguish between odd and even lengths), or use [np.median](https://docs.scipy.org/doc/numpy/reference/generated/numpy.median.html) from Numpy.

Reply:
I'll fix it, my goal was to remove pandas/numpy dependencies to hopefully be able to run archery without pip install.